lib/routes/token.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var validators = require('./validators')
+var HEX_STRING = validators.HEX_STRING
+
+module.exports = function (log, isA, error, db) {
+
+  var routes = [
+    {
+      method: 'POST',
+      path: '/token/verify',
```

Comment:
IIUC, current plan is for the initial version to not use this new endpoint?  If so then perhaps we can just remove it entirely for now.